engines/alg/detection.cpp

```diff
@@ -0,0 +1,69 @@
+/* ScummVM - Graphic Adventure Engine
+ *
+ * ScummVM is the legal property of its developers, whose names
+ * are too numerous to list here. Please refer to the COPYRIGHT
+ * file distributed with this source distribution.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+#include "base/plugins.h"
+#include "engines/advancedDetector.h"
+
+#include "alg/alg.h"
+#include "alg/detection.h"
+#include "alg/detection_tables.h"
+
+static const PlainGameDescriptor algGames[] = {
+	{ "cpatrol", "Crime Patrol" },
+	{ "dwars", "Drug Wars" },
```

Comment:
```suggestion
	{ "dwars", "Crime Patrol 2: Drug Wars" },
```
I believe that this is the full game name

Reply:
fixed